data/translations/zh/state-data.json

```diff
@@ -0,0 +1,469 @@
+  },
+  "as": {
+    "hp_link": "https://aselectionoffice.gov/",
+    "more_info_link": "https://aselectionoffice.gov/",
+    "ip_deadline": "选举日前30 天",
+    "confirm_registration_link": "https://aselectionoffice.gov/status.php"
+  },
+  "az": {
+    "hp_link": "https://azsos.gov/elections",
+    "registration_link": "https://servicearizona.com/VoterRegistration/selectLanguage",
+    "more_info_link": "https://azsos.gov/elections/voting-election/register-vote-or-update-your-current-voter-information",
+    "online_deadline": "选举日前 29 天",
+    "mail_postmarked_deadline": "必须在选举日前29 天盖上邮戳",
+    "ip_deadline": "选举日前 29 天",
+    "confirm_registration_link": "https://my.arizona.vote/WhereToVote.aspx?s=individual"
+  },
+  "ar": {
+    "hp_link": "https://www.sos.arkansas.gov/elections/",
+    "more_info_link": "https://www.sos.arkansas.gov/elections/voter-information/",
+    "ip_deadline": "选举日前 30 天",
+    "mail_postmarked_deadline": "必须在选举日前 30 天盖上邮戳",
+    "confirm_registration_link": "https://www.voterview.ar-nova.org/voterview"
+  },
+  "ca": {
+    "hp_link": "https://www.sos.ca.gov/elections",
+    "registration_link": "https://registertovote.ca.gov/",
+    "more_info_link": "https://www.sos.ca.gov/elections/voting-resources/voting-california/registering-vote",
+    "ip_deadline": "直到选举日 (包括选举日) 都可以使用",
+    "online_deadline": "选举日前 15 天",
+    "mail_postmarked_deadline": "必须在选举日前 15 天盖上邮戳",
+    "confirm_registration_link": "https://registertovote.ca.gov/"
+  },
+  "co": {
+    "hp_link": "https://www.sos.state.co.us/pubs/elections/",
+    "registration_link": "https://www.sos.state.co.us/voter/pages/pub/home.xhtml",
+    "more_info_link": "https://www.sos.state.co.us/pubs/elections/vote/VoterHome.html",
+    "ip_deadline": "直到选举日 (包括选举日) 都可以使用",
+    "online_deadline": "选举日前 8 天",
+    "mail_postmarked_deadline": "必须在选举日前8天盖上邮戳",
+    "confirm_registration_link": "https://www.sos.state.co.us/voter/pages/pub/olvr/findVoterReg.xhtml"
+  },
+  "ct": {
+    "hp_link": "https://portal.ct.gov/sots",
+    "registration_link": "https://voterregistration.ct.gov/OLVR/welcome.do",
+    "more_info_link": "https://portal.ct.gov/SOTS/Election-Services/Voter-Information/Voter-Fact-Sheet",
+    "ip_deadline": "直到选举日 (包括选举日) 都可以使用",
+    "online_deadline": "选举日前7 天",
+    "mail_postmarked_deadline": "必须在选举日前 7 天盖上邮戳",
+    "confirm_registration_link": "https://portaldir.ct.gov/sots/LookUp.aspx"
+  },
+  "de": {
+    "hp_link": "https://elections.delaware.gov/index.shtml",
+    "registration_link": "https://ivote.de.gov/VoterView/",
+    "more_info_link": "https://elections.delaware.gov/voter/votereg.shtml",
+    "ip_deadline": "选举日前 第四个星期六",
+    "online_deadline": "选举日前 第四个星期六",
+    "mail_postmarked_deadline": "必须在选举日前第四个星期六盖上邮戳",
+    "confirm_registration_link": "https://ivote.de.gov/VoterView"
+  },
+  "dc": {
+    "hp_link": "https://dcboe.org/",
+    "registration_link": "https://forms.dc.gov/f/DC_Voter_Registration_Form",
```

Comment:
Update registration_link to `https://www.dcboe.org/Voters/Register-To-Vote/Register-to-Vote`